menpo/feature/visualize.py

```diff
@@ -25,3 +27,86 @@ def sum_channels(pixels, channels=None):
     else:
         sum_image = np.sum(pixels[channels], axis=0)
     return sum_image.reshape((1,) + sum_image.shape)  # add a channel axis
+
+
+# TODO: Needs fixing ...
```

Comment:
Can we just re-remove this? I removed it from menpowidgets already

Reply:
> There's a cpp file that seems to have been accidentally committed that needs to be removed. If we can take a lot of the other changes (some of which look super useful) into new PRs then we can discuss the visualization more easily here?

Agree. I assume I  have to cherry pick the commits for the new PRs, merge them and afterwards I can create a new 'clean' commit with the visualization staff onluy